db/util_testing.go

```diff
@@ -206,11 +204,7 @@ func purgeWithDCPFeed(ctx context.Context, dataStore sgbucket.DataStore, tbp *ba
 
 		key := string(event.Key)
 
-		if base.TestUseXattrs() {
-			purgeErr = dataStore.DeleteWithXattrs(ctx, key, []string{base.SyncXattrName})
-		} else {
-			purgeErr = dataStore.Delete(key)
-		}
+		purgeErr := dataStore.DeleteWithXattrs(ctx, key, []string{base.SyncXattrName})
```

Comment:
This confused me at first but does make sense since `DeleteWithXattrs` will handle both xattr and non-xattr states of documents the same way.

Might be worth a comment?

Reply:
Added a comment for this.